src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/TelemetryUtility.cs

```diff
@@ -25,6 +25,16 @@ public static string CreateFileAndForgetEventName(string typeName, string member
             return $"{VSTelemetrySession.VSEventNamePrefix}fileandforget/{typeName}/{memberName}";
         }
 
+        public static void EmitException(string className, string methodName, Exception exception)
+        {
+            TelemetryEvent telemetryEvent = new TelemetryEvent($"errors/{className}.{methodName}");
+            telemetryEvent["Message"] = exception.Message;
+            telemetryEvent["ExceptionType"] = exception.GetType().FullName;
```

Comment:
I'd consider unwrapping the exception to get the inner most one.
Maybe report both?